scripts/jest/setup.js

```diff
@@ -32,5 +32,7 @@ jest.doMock('react', () => {
   }
   return mockActualReact;
 });
-global.TextEncoder = TextEncoder; 
+global.TextEncoder = TextEncoder;
 registerRequireContextHook();
+
+jest.mock('../../packages/bpk-react-utils/src/deferCallback');
```

Comment:
Globally mock deferCallback to ensure calendar snapshots include real aria label on initial render, not 'loading...' fallback.